### DISCUSSION TITLE
Non-dicom workflows powered by FHIR & HL7

### DISCUSSION THREAD

**Discussion by hshuaib90** [Project-MONAI/monai-deploy#67] [Ideas]
With the great progress we've made with Informatics Gateway and Workflow Manager it is time to thoroughly test that workflow. This means that there will likely be a pause with releases of the IG, as it is relatively feature complete in its current incarnation.

As a reminder, the roadmap for the MONAI Deploy WG, among other things, is aiming to:

> - Radiology first, then other modalities like Pathology.
> - Interoperability with clinical systems. Starting with DICOM, then FHIR.

We are now at the stage where we should seriously consider how we intend to support HL7 & FHIR workflows (in combination with DICOM and standalone).

In order to kick this off, we should begin to compile use-cases, products etc. that leverage non-dicom information. For example: https://www.cibiltech.com/ibox

Can we use this discussion topic to add to the list please? @laurencejackson etc.

**Comment by mocsharp** [comment]
RESERVED!

**Comment by mocsharp** [comment]
RESERVED!
